ui/lib/core/addon/components/form-field.hbs

```diff
@@ -80,8 +83,54 @@
           {{/if}}
         </Hds::Form::Select::Field>
       {{/if}}
+    {{else}}
+      {{#if (or (eq @attr.type "number") (eq @attr.type "string"))}}
```

Comment:
Putting a note here for posterity as I missed this at first glance. "number" and "string" are still handled in the original form-field logic. Because of [this](https://github.com/hashicorp/vault/pull/29962/files#diff-b4ed6df7629e08ec0f436921d541c746b69d27e80a43f54844e800e6a9f17867R102) line, they are not caught in this logic.

Reply:
Discussed it with @Monkeychip, it's all good.